File: Web/src/app/views/reports/Models/ReportSearchCriteria.ts
import { BaseSearchCriteria } from '../../catalog/Models/BaseSearchCriteria'
export class ReportSearchCriteria extends BaseSearchCriteria {
    shiftId: number = 0
    refundTypeId: number = 0
    discountKeyId: number = 0
    isComplimentary: boolean
}

File: Back-End/QAFastTrack.Core/SearchCriteria/ReportSearchCriteria.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Restaurant.Core.SearchCriteria
{
    public class ReportSearchCriteria: BaseSearchCriteria
    {
        public string? UserId { get; set; }
        public int RefundTypeId { get; set; }
        public int ShiftId { get; set; }
        public int DiscountKeyId { get; set; }
        public bool IncludeComplimentary { get; set; }
    }
}
